packages/cli/templates/webcomponents/igc-ts/projects/_base/files/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en-GB">
+
+<head>
+  <meta charset="utf-8">
+  <base href="/">
+  <link rel="stylesheet" href="./node_modules/igniteui-webcomponents/themes/bootstrap.css">
```

Comment:
The `index.html` in the empty proj is using themes/**light**/bootstrap.css, I'm assuming that's updated from the new package. This one should be too, right? Otherwise it won't work